src/styles/product-card.css

```diff
@@ -0,0 +1,94 @@
+.product-card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+  width: 200px;
+
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.product-card__image {
+  display: block;
+
+  margin-bottom: 40px;
+  width: 160px;
+  height: 134px;
+}
+
+.product-card__product-name {
+  margin-bottom: 4px;
+
+  font-size: 12px;
+  line-height: 18px;
+  font-weight: 500;
+}
+
+.product-card__code {
+  align-self: flex-start;
+
+  margin-bottom: 16px;
+
+  color: var(--font-color-light);
+}
+
+.product-card__feedback {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  margin-bottom: 24px;
+  width: 100%;
```

Comment:
redundant
```suggestion
```

Reply:
But if this element doesn't have its width, justify-content: space-between can't put its daughters' elements as we want to have.